daisy_workflows/export/export_sbom.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
You are adding the same file in 2 different pull requests?

Reply:
yes